src/v/kafka/client/brokers.cc

```diff
@@ -95,4 +95,10 @@ ss::future<bool> brokers::empty() const {
     return ss::make_ready_future<bool>(_brokers.empty());
 }
 
+ss::future<> brokers::clear() {
```

Comment:
Majority of this commit is adding `clear()` methods. I'll split these up in the next push so the commit diffs are not as large.